.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install dependencies and set path
+        run: |
+            sudo apt-get update
+            sudo apt-get install -y ninja-build libcunit1-dev  python3-pip
+            # Install meson as root so we can install to the system below.
+            sudo pip install meson
+      - name: Build tarball
+        run: |
+          cd c
+          rm test-data
+          cp -r ../test-data .
+          meson . ../build-gcc
+          cd ../build-gcc
+          git config --global user.email "CI@CI.com"
+          git config --global user.name "Mr Robot"
+          git add -A
+          git commit -m "dummy commit to make meson add in the symlinked directory"
+          meson dist
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        if: startsWith(github.ref, 'refs/tags/')
```

Comment:
Shouldn't we filter by the tag name here, like starts with "C_"? Otherwise, won't we post tarballs for Python releases as well?

Where do the artefacts get pushed to pypi?

Reply:
> Shouldn't we filter by the tag name here, like starts with "C_"? Otherwise, won't we post tarballs for Python releases as well?

Sure, I thought it might still be useful so that you can just fetch from the latest release, but people can filter. Fixed.

> Where do the artefacts get pushed to pypi?

In the separate `wheels.yml` action.